src/test/resources/features/integration/health-check.feature

```diff
@@ -24,4 +24,4 @@ Feature: Tests the availability of the Microservice
   Scenario: checks if the metrics endpoint is up
     Given path 'actuator/metrics'
```

Comment:
i think that we are trying to standardize to use this separated by commas and not using the slash, right?